docs/quickstarts/sso-quickstart-customize.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Customize your Node.js SSO-enabled add-in
 description: Learn about customizing the SSO-enabled add-in that you created with the Yeoman generator.
-ms.date: 02/20/2020
+ms.date: 07/30/2020
 ms.prod: non-product-specific
 localization_priority: Normal
 ---
```

Comment:
Is there supposed to be a change in this file?

Reply:
good catch